recipes/lucit-licensing-python/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "lucit-licensing-python" %}
+{% set version = "1.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/LUCIT-Systems-and-Development/lucit-licensing-python/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 49eb84c50772d25e8723ac8d39c7cbae29ab90215d173d7b84fd1c2440fae9c7
+
+build:
+  skip: true  # [py<37]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - setuptools
+    - wheel
+    - cython
+    - pip
+  run:    
+    - python
+    - cython
```

Comment:
Is cython really needed at run time?

Reply:
Yes! Its loaded and used in runtime.

Reply:
https://github.com/LUCIT-Systems-and-Development/lucit-licensing-python/blob/8061f780547743c820631987652f17a8c8642857/lucit_licensing_python/manager.py#L19